src/github.com/getlantern/flashlight/app/mock_autoupdate.go

```diff
@@ -0,0 +1,15 @@
+// +build mockupdate
+
+package app
+
+// This is a public key derived from a private key that is freely available.
+// Don't use in production.
+const packagePublicKey = `-----BEGIN PUBLIC KEY-----
```

Comment:
Hmm, we do use `packagePublicKey` in different places:

```
grep -R packagePublicKey .
./github.com/getlantern/flashlight/app/mock_autoupdate.go:const packagePublicKey = `-----BEGIN PUBLIC KEY-----
./github.com/getlantern/flashlight/app/app.go:  autoupdate.PublicKey = []byte(packagePublicKey)
./github.com/getlantern/flashlight/app/autoupdate.go:const packagePublicKey = `-----BEGIN PUBLIC KEY-----
```

And, keys are also different:

```
vagrant@lantern:~/go/src/github.com/getlantern/lantern-devel/src$ cat ./github.com/getlantern/flashlight/app/mock_autoupdate.go
// +build mockupdate

package app

// This is a public key derived from a private key that is freely available.
// Don't use in production.
const packagePublicKey = `-----BEGIN PUBLIC KEY-----
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzSoibtACnqcp2uTGjCMJ
tTOLDIMQ4oGPhGHT4Q/epum+H3hcbBNs9jRnMRWgX4z++xxuNJnhmoJw0eUXB7B4
vj5DYpPajq6gPY8JuraF4ngfP5oxKj2BqpEUR9bx+3SjOSInrirM0JZO+aAW38BQ
NJB+sS7JvbPjcwdjwKc5IKzc9kxxJNoZoFE9GMnYzaOrAlpCuAKWH8SCXYtCTxsX
fKexdDxsI5Vzm5lQHJLMeqhLTQTUm9oQofwNAOGOkn6dD4ObMlmFTOsf1G03/Dl9
sVgjaWaZ9bGjvJ9B85UxNeWwduy+uMrqFytxG6bbq0PbDEVu6ZQCPyiyCA7l945J
OQIDAQAB
-----END PUBLIC KEY-----`
vagrant@lantern:~/go/src/github.com/getlantern/lantern-devel/src$ cat ./github.com/getlantern/flashlight/app/autoupdate.go
// +build !mockupdate

package app

// This is the public key of the BNS cert. Incoming updates will be signed to
// prevent MITM attacks.
const packagePublicKey = `-----BEGIN PUBLIC KEY-----
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxeReZ0VHDQ+/XYEHhFq0
krT+a/+/mlhCkgJ/605KmPXqBv8qo5f1iK6C+TQ87264J4Z9yw0tRwcdY1/ofpH7
Tywq3pBOgfrnnP9gFtquQ/tgzVkorQ0L51w9HLZ3cCjpaLpofIaztgbCIzsCT6kV
Nx6Sd/4KBSuThhMEnP5pu5Wxr4/lujIpTeVEXzljQMxqX+58ISeXYx6SxLXx5Vgj
1IB6NJwjg7r4Nzg/zUH0ZkCWj3rDWo6itIoeo61o+hPQAjH23TCKOn8Ssaejocyg
CrcOc7aqfGuVM3HuHxtXsjYPqJMVHiXKosi9HcHo5BACPT0FkrZIwz3k6Vy1h7nB
HQIDAQAB
-----END PUBLIC KEY-----`
```